doc/src/errorrecovery.md

```diff
@@ -253,7 +253,21 @@ few lines of code.
 ## Making use of `%epp` for easier to read repair sequences
 
 By default, pretty-printing lexeme types prints out their identifier in the
-grammar. These rarely match what the user would expect:
+grammar. To this point these have matched the output the user would expect,
```

Comment:
Do we need to repeat this? I think this file has the lexer definition earlier on in it?

Reply:
The lexer definitions all refer to the initial `calc.l` definition from the quickstart.md,
which now specify '*' instead of 'MUL', but this section explicitly needs 'MUL', etc.
So that it can be renamed '*' by `%epp MUL "*"`, substituting textual names for symbols during error recovery.

So this section kind of needs some textual name to motivate the example yes.

Reply:
I tried to clarify this text in [892cc71](https://github.com/softdevteam/grmtools/pull/366/commits/892cc71bbbae9eb24f9da69a547b80167198ef1d) to explain the motivation for this extra lex definition here.